src/main/java/com/minecolonies/coremod/colony/requestsystem/resolvers/StandardPlayerRequestResolver.java

```diff
@@ -80,6 +85,30 @@ public void resolve(@NotNull final IRequestManager manager, @NotNull final IRequ
         final IColony colony = manager.getColony();
         if (colony instanceof Colony)
         {
+            if (Configurations.requestSystem.creativeResolve &&
+                  request.getRequest() instanceof IDeliverable &&
+                  request.getRequester() instanceof BuildingBasedRequester &&
+                  ((BuildingBasedRequester) request.getRequester()).getBuilding().isPresent() &&
+                  ((BuildingBasedRequester) request.getRequester()).getBuilding().get() instanceof AbstractBuilding)
+            {
+                AbstractBuilding building = (AbstractBuilding) ((BuildingBasedRequester) request.getRequester()).getBuilding().get();
+                Optional<CitizenData> citizenDataOptional = building.getCitizenForRequest(request.getToken());
+
+                List<ItemStack> resolvablestacks = request.getDisplayStacks();
+                if (!resolvablestacks.isEmpty() && citizenDataOptional.isPresent())
+                {
+                    ItemStack resolveStack = resolvablestacks.get(0);
+                    resolveStack.setCount(Math.min(((IDeliverable) request.getRequest()).getCount(), resolveStack.getMaxStackSize()));
+                    ItemStack remainingItemStack = InventoryUtils.addItemStackToItemHandlerWithResult(new InvWrapper(citizenDataOptional.get().getCitizenEntity().getInventoryCitizen()), resolveStack);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 200 characters long line (which is greater than 180 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00103)